src/System.Management.Automation/engine/remoting/server/ServerRunspacePoolDriver.cs

```diff
@@ -1392,126 +1408,106 @@ private class DebuggerCommandArgument
                 // Input parameters:
                 // [-Id <int>]
                 // Returns Breakpoint object(s).
-                string script = null;
+                // at this time.
```

Comment:
Don't understand the importance of this comment

Reply:
typo. deleted.